stacker/providers/aws/interactive.py

```diff
@@ -80,7 +80,7 @@ def output_summary(fqn, action, changeset, replacements_only=False):
     changes = []
     for change in changeset:
         resource = change['ResourceChange']
-        replacement = resource['Replacement'] == 'True'
+        replacement = resource.get('Replacement', '') == 'True'
```

Comment:
Only a minor nit: You can just do `resource.get('Replacement') == 'True'`

`.get()` will return `None` by default if the key doesn't exist.